JS/Lesson22/tugba/style.css

```diff
@@ -0,0 +1,39 @@
+body {
+    font-family: Arial, sans-serif;
+}
+
+.accordion-container {
+    width: 60%;
+    max-width: 600px;
+}
+
+.accordion {
+    display: flex;
+    justify-content: center;
+    background-color: #f9f9f9;
+    color: #444;
+    cursor: pointer;
+    padding: 18px;
+    width: 80%;
```

Comment:
I'd make the button 100% width of it's parent element. However, if you apply the suggestion above to make .`accordion__container` a flex container, then you don't need the width at all here.